SPECS/python-packaging/python-packaging.spec

```diff
@@ -55,6 +55,9 @@ Core utilities for Python packages
 %license LICENSE
 
 %changelog
+* Thu Aug 29 2024 Andrew Phelps <anphel@microsoft.com> - 23.2-3
+- Bump release to rebuild and resolve python3dist provides issue
```

Comment:
We'd need the same bump for `python-pygments` as they are also built before `python-packaging` is installed during the toolchain build.

Reply:
Thanks, I've updated this